Apps/Common/src/Delegates/RestImmunizationDelegate.cs

```diff
@@ -76,6 +76,38 @@ public class RestImmunizationDelegate : IImmunizationDelegate
             this.immunizationEndpoint = $"{endpoint}{path}";
         }
 
+        /// <inheritdoc/>
+        public Task<RequestResult<IList<ImmunizationEvent>>> GetCovidImmunization(string phn)
```

Comment:
Does the mock data used here need to be replaced as part of this PR?

Reply:
this is no longer necessary since we are using the admin delegate